### PR TITLE
[PT FE] Support multiple FX operations

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Unpack nested tuples at model outputs*
 - *Support for `aten.cat.default` with default axis*
 - *Support `aten::reciprocal_` and `aten::abs_` for TorchScript case*
 - *Support:*
    *  `aten.adaptive_max_pool3d.default`
    *  `aten.adaptive_max_pool{N}d.default`
    *  `aten.amin.default`
    *  `aten.argmin.default`
    *  `aten.bitwise_{XXX}.default`
    *  `aten.clamp_max.default`
    *  `aten.clamp_{max/min}.Tensor`
    *  `aten.fill.Tensor`
    *  `aten.flip.default`
    *  `aten.fmod.{Scalar/Tensor}`
    *  `aten.ge.{Scalar/Tensor}`
    *  `aten.gt.Tensor`
    *  `aten.index_select.default`
    *  `aten.le.{Scalar/Tensor}`
    *  `aten.log{10/1p/2}.default`
    *  `aten.{max/maximum/mean/min/minimum}.default`
    *  `aten.min.dim`
    *  `aten.ne.Tensor`
    *  `aten.pow.{Scalar/Tensor_Tensor}`
    *  `aten.reciprocal.default`
    *  `aten.rsub.Tensor`
    *  `aten.scatter.value`
    *  `aten.select_scatter.default`
    *  `aten.sign.default`
    *  `aten.sqrt.default`
    *  `aten.sum.default`
    *  `aten.unfold.default`
    *  `aten.var.correction`
    *  *trigonometric ops*

 - *Enabled layer tests for added ops for torch.export case*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_scatter.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def _forward_inplace_reduce(self, x: torch.Tensor):
     @pytest.mark.parametrize("dtype", ["int32", "int64", "float32", "float64"])
     @pytest.mark.parametrize(["inplace", "has_out"], [(True, False), (False, True), (False, False)])
     @pytest.mark.parametrize("reduce", [None, "add", "multiply"])
-    def test_scatter(self, dim, index, src, dtype, inplace, has_out, reduce, ie_device, precision, ir_version):
+    def test_scatter1(self, dim, index, src, dtype, inplace, has_out, reduce, ie_device, precision, ir_version):
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_scatter.py:107]:
```suggestion
    def test_scatter(self, dim, index, src, dtype, inplace, has_out, reduce, ie_device, precision, ir_version):
```

File: tests/model_hub_tests/pytorch/test_timm.py [hunk 2]
```diff
@@ -86,7 +89,7 @@ def test_convert_model_precommit(self, name, ie_device):
         self.run(name, None, ie_device)
 
     @pytest.mark.nightly
-    @pytest.mark.parametrize("mode", ["trace"]) # disable "export" for now
+    @pytest.mark.parametrize("mode", ["export"]) # disable "export" for now
```
Comment (mvafin) [inline tests/model_hub_tests/pytorch/test_timm.py:92]:
```suggestion
    @pytest.mark.parametrize("mode", ["trace"]) # disable "export" for now
```

File: tests/model_hub_tests/pytorch/test_timm.py [hunk 3]
```diff
@@ -18,10 +18,13 @@
 def filter_timm(timm_list: list) -> list:
     unique_models = dict()
     filtered_list = []
-    ignore_list = ["base", "xxtiny", "xxs", "pico", "xtiny", "xs", "nano", "tiny", "s", "mini", "small", "lite",
-                   "medium", "m", "big", "large", "l", "xlarge", "xl", "huge", "xxlarge", "gigantic", "giant", "enormous"]
+    ignore_list = ["base", "atto", "femto", "xxtiny", "xxsmall", "xxs", "pico", "xtiny", "xmall", "xs", "nano", "tiny", "s", "mini",
+                   "small", "lite", "medium", "m", "big", "large", "l", "xlarge", "xl", "huge", "xxlarge", "gigantic", "giant", "enormous"]
```
Comment (mvafin) [inline tests/model_hub_tests/pytorch/test_timm.py:22]:
```suggestion
    ignore_list = ["base", "xxtiny", "xxs", "pico", "xtiny", "xs", "nano", "tiny", "s", "mini", "small", "lite",
                   "medium", "m", "big", "large", "l", "xlarge", "xl", "huge", "xxlarge", "gigantic", "giant", "enormous"]
```

File: src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py [hunk 4]
```diff
@@ -144,13 +157,13 @@ def get_input_strides(self, index: int) -> list:
...
-            return OVAny(pt_to_ov_type_map[str(self.input_types[index])])
+            return self.input_types[index]
         input = self._raw_input(index)
         return self.get_type_for_value(input)
 
     def get_output_debug_name(self, index):
-        if self._output_names is not None and index < len(self._output_names):
-            return self._output_names[index]
+        if self._outputs is not None and index < len(self._outputs) and self._outputs[index][0]:
+            return self._outputs[index][0]
```
Comment (cavusmustafa) [inline src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:166]: For non dictionary outputs, will this return an empty string?

Comment (mvafin) [reply src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:166]: good catch, yes it will, need to check that it is not empty string

Comment (mvafin) [reply src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:166]: `self._outputs[index][0]` check that name is empty or not.

File: src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py [hunk 5]
```diff
@@ -83,6 +78,24 @@ def inputs(self):
...
+        if isinstance(arg, (tuple, list)):
+            res = []
+            for e in arg:
+                res.extend(TorchFXPythonDecoder.unpack_containers(e))
+            return res
+        elif isinstance(arg, dict):
+            res = []
+            for k, e in arg.items():
+                unpacked = TorchFXPythonDecoder.unpack_containers(e)
+                if len(unpacked) == 1:
```
[the remark this answers is not in the document]
Comment (mvafin) [reply src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:92]: Yes it will. If dict contain tuple with more then 1 element the name will be lost

**Review by mryzhov — APPROVED** [review]

**Review by cavusmustafa — APPROVED** [review]
